src/main/java/thePackmaster/patches/CompendiumPatches.java

```diff
@@ -67,7 +72,7 @@ public int[] Locate(CtBehavior ctMethodToPatch) throws Exception {
     public static class CustomOrdering {
         @SpirePostfixPatch
         public static void catchSort(CardLibSortHeader __instance, SortHeaderButton button, boolean isAscending) {
-            if (button == packButton) {
+            if (button == packButton && RenderBaseGameCardPackTopTextPatches.isInPackmasterCardLibraryScreen()) {
```

Comment:
I believe that since you can't click the sort by packs button on other tabs, this check may be unnecessary (since we can assume that when you hit this code, you're on the Packmaster tab). I still thought it was prudent to have the check here too, as an extra safeguard in case we missed some way of hitting this, and to make it clear that the intent of this is to only work for the Packmaster tab.